contentcuration/contentcuration/frontend/channelEdit/components/QuickEditModal/EditBooleanMapModal.vue

```diff
@@ -26,9 +31,9 @@
       :inputHandler="(value) => { selectedValues = value }"
     ></slot>
 
-    <span v-if="error" class="red--text">
+    <!-- <span v-if="error" class="red--text">
       {{ error }}
-    </span>
+    </span> -->
```

Comment:
I think this is no longer needed at all with the change that you've made, so let's delete rather than just comment it out